Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -174,20 +174,28 @@ class AppStoreConnectService {
         firstName: String?,
         lastName: String?,
         inviteType: BetaInviteType?,
-        appIds: [String],
-        bundleIds: [String],
+        filterIdentifiers: [AppLookupIdentifier],
         groupNames: [String],
         sort: ListBetaTesters.Sort?,
         limit: Int?,
         relatedResourcesLimit: Int?
     ) throws -> [BetaTester] {
 
-        var appIds: [String] = appIds
-        if !bundleIds.isEmpty && appIds.isEmpty {
-            appIds = try GetAppsOperation(options: .init(bundleIds: bundleIds))
-                .execute(with: requestor)
-                .await()
-                .map(\.id)
+        var filterAppIds: [String] = []
+        var filterBundleIds: [String] = []
+
+        _ = filterIdentifiers.map { identifier in
+            switch (identifier) {
```

Comment:
```suggestion
        filterIdentifiers.forEach { identifier in
            switch identifier {
```

Reply:
Is there any extra benefit of using forEach over map?

Reply:
I think performance wise both are same only map is returning an array which we don't need but for each doesn't..so I will use foreach as you have suggested..let me know if there is any performance benefit .